src/index.html

```diff
@@ -5,8 +5,396 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo">
+          <img src="images/logo.png"
+          alt="Logo"
+          class="top-bar__image"
+          >
```

Comment:
<img width="357" alt="attrCheklist" src="https://github.com/mate-academy/layout_catalog/assets/77551820/44d5b979-8cf1-453e-adfa-12cf2b969c77">

**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**